### DISCUSSION TITLE
CRC book example

### DISCUSSION THREAD

**Discussion by bgreenwell** [quarto-dev/quarto-cli#3407] [Q&A]
Would it be possible to include an example of using Quarto to write a CRC-style book? Are there any examples? Similar in spirit to [this CRC example](https://github.com/yihui/bookdown-crc) from bookdown which shows how to use the `krantz.cls` style with appropriate `.tex` supplementary files.

**Comment by mcanouil** [comment]
I am not aware of a CRC book published using Quarto.
Did you take a look at the gallery? <https://quarto.org/docs/gallery/#books>

The LaTeX template is not much more different than articles templates, thus you could also take a look at <http://github.com/quarto-journals> and the extension documentation <https://quarto.org/docs/extensions/>.

**Comment by baptiste** [comment]
I'm working on one; following Yihui's Bookdown workflow and adjusting it to Quarto seems to work mostly well so far. I did reach out to their technical team at some point, asking exactly that ("Do you guys have a how-to/example of using Quarto, similar to Yihui's for Bookdown?") and it was utterly useless. First response was "no, this is not possible because our workflow uses LaTeX", and when I pointed out that this was exactly the same situation as with Bookdown, they asked _me_ to provide them with an example ¯\_(ツ)_/¯, so I gave up on that conversation. 
The sample chapters I've sent so far to the Editor seem to pass their quality checks, so I don't think there'll be too many problems (worst case scenario, I'll have to tweak a bit the final TeX).

**Reply by bgreenwell** [reply]
Thanks @baptiste, I was able to follow the example and get it to work, albeit with a minor issue of the front and back matter appearing twice in the ToC. So I’ll probably do some tweaking of the final latex document as well!

**Comment by bgreenwell** [comment]
For those interested, I started a repo with a minimal-ish example here: https://github.com/bgreenwell/quarto-crc; similar in spirit to @yihui's [bookdown-crc](https://github.com/yihui/bookdown-crc) example. Only issue I'm still having is the double ToC entries for both the front and back matter.

**Reply by iangow** [reply]
Did you ever sort out the double-ToC entries?

**Comment by friendly** [comment]
@baptiste I'm working on the final stages of my book for CRC press, and just ran into an inexplicable change in the font size for the whole book, https://tex.stackexchange.com/questions/751552/main-font-size-has-changed-in-pdf-and-i-cant-see-where-or-how-to-fix-this

Did you do anything special in your repo to make the `krantz.cls` work for you as you wanted?

**Reply by baptiste** [reply]
Sorry I ditched CRC after they gave authors the middle finger – my last interaction with them was this letter:

<details>

Dear Publisher,

Let me recapitulate:
 
- T&F did not contact us about their dealings with GenAI companies (https://pivot-to-ai.com/2024/07/20/taylor-francis-feeds-academics-work-to-microsoft-ai/); I learned it in the news. I can imagine that T&F did not feel particularly proud of it and hoped it'd go unnoticed
 
- There is no opting out for authors
 
- It's presented as a Good Thing with no acknowledgment of concerns
 
- Explanations immediately turn to regurgitation the official "AI position as a company", which to be honest might just as well be written by a LLM
 
I am sorry that T&F has taken this path. It is the wrong one. And I will have nothing to do with any of this.
 
In case you're quick to think that I'm being unnecessarily difficult and retrograde against this inevitable new wave of technology, I will just list a few reasons I choose to take a clear stance against the genAI industry in its current form. I like to think that any of these reasons should really make anyone pause and question what we're allowing to happen at the moment.
 
- The entire industry would not exist without the gigantic theft of intellectual property by a few companies that consider themselves too big to care about the law (compare that to Aaron Swartz's story). There is no ethical LLM in existence at present.
 
- The insane carbon footprint annihilating any recent efforts to curb climate change
 
- The psychological trauma and disgusting practices for "AI alignment" on Kenyan workers, exposed for hours on end to the worst content humanity has produced, to shield western eyes from its ugly underbelly
 
- The sordid connection between techno-bro leaders in this industry and one version or another of TESCREAL eugenics fantasies
 
- The complete disregard for any societal consequences, e.g. is it actually safe for our children? Good for their education? For our mental health? For democracy, for society? None of the normal precautionary principles seem to apply when you have that much money and can easily distract people with doomsday red-herrings or Wall-street drama.
 
I could go on, but really, anyone who needs more reasons to turn away has probably already gone too far outsourcing their thinking to chatGPT. I am sorry your company is taking this unfortunate stance; I do hope it's a temporary mistake and that they will walk back soon (ideally not just for economic reasons). 
 
</details>

Before then, I was particularly unimpressed with the state of their latex workflow (and lack of understanding, let alone support). I did find a few tricks on my own by making a dummy project with just a very minimal set of files and turning on all intermediate debugging files (keep-tex: true and the like).
I suspect you're facing a change in quarto or pandoc that is no longer passing the option downstream at some point or another because of a syntax change or something like that.
